.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
this seems like it doesn't belong in the repo :)

Reply:
I'm not sure to understand what's the problem? This clause means the editor has to stop while looking for a `.editorconfig` in parent directories of its working directory:
```
When opening a file, EditorConfig plugins look for a file named .editorconfig in the directory of the opened file and in every parent directory. A search for .editorconfig files will stop if the root filepath is reached or an EditorConfig file with root=true is found.
``` 
(source: https://editorconfig.org/#file-location)